base_report_to_printer/data/neutralize.sql

```diff
@@ -0,0 +1,2 @@
+update printing_printer set active=false;
```

Comment:
```suggestion
# pylint: disable=file-not-used
update printing_printer set active=false;
```

Reply:
file changed!

Reply:
Maybe the pylint exception has to be put in another place... @moylop260 do you know?

Reply:
Hi everybody

The pylint-odoo version used is too old:
 - https://github.com/OCA/report-print-send/blob/af5df33b010dc671ca33a6b4d4fc0867a7718ae2/.pre-commit-config.yaml#L128

The latest one is https://pypi.org/project/pylint-odoo/8.0.19

@yajo updated the template in order to update the pylint configuration file too

Regards!

Reply:
> Hi everybody
> 
> The pylint-odoo version used is too old:
>  - https://github.com/OCA/report-print-send/blob/af5df33b010dc671ca33a6b4d4fc0867a7718ae2/.pre-commit-config.yaml#L128
> 
> The latest one is https://pypi.org/project/pylint-odoo/8.0.19
> 
> @yajo updated the template in order to update the pylint configuration file too
> 
> Regards!

Hello, now what should i do? The yaml in my PR Is old. How can i change the head of the PR?

Reply:
First, we need to merge https://github.com/OCA/oca-addons-repo-template/pull/198. Then do a template release and update the template here. Then, you can rebase the PR and all will get green.

Reply:
@yajo the error persists...